src/components/LoanCards/BorrowerInfo/BorrowerInfoHeaderExpandable.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+	<div class="borrower-info-header-expandable" :class="{expanded}">
+		<div class="name" v-if="name">{{ name }}</div>
+		<div class="details" v-if="country || activity">
+			{{ country ? `${country}` : '' }}
+			{{ country && activity ? '|' : '' }}
+			{{ activity ? `${activity}` : '' }}
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		country: {
+			type: String,
+			default: '',
+		},
+		activity: {
+			type: String,
+			default: '',
+		},
+		loanId: {
+			type: Number,
+			default: null,
+		},
+		name: {
+			type: String,
+			default: '',
+		},
+		expanded: {
+			type: Boolean,
+			default: false,
+		},
+	},
```

Comment:
Since this component is only using props, it can be a functional component, which lets you drop the script tag entirely. See https://vue-loader.vuejs.org/guide/functional.html

Reply:
Also, functional components render faster and have a smaller compiled footprint, so it's good to use them when possible.

Reply:
Good call :+1: will be fixed in next commit